schema.graphql

```diff
@@ -31,6 +31,12 @@ enum PoolState {
   Stale
 }
 
+enum PoolType {
+  GeyserV1
+  GeyserV2
+  Fountain
+}
+
```

Comment:
I think, at least for now, its more helpful to have PoolType since it's more aligned with what the user sees/cares about. Also, determining what to display client side isn't cleanly separated between staking module and reward module, since the stake pane display is actually determined by the reward module, rather than staking module.

Reply:
Makes sense, let's revisit this after the v2 launch